gdk4/src/auto/content_provider.rs

```diff
@@ -118,22 +115,6 @@ impl<O: IsA<ContentProvider>> ContentProviderExt for O {
         }
     }
 
-    fn value(&self, value: &mut glib::Value) -> Result<(), glib::Error> {
-        unsafe {
-            let mut error = ptr::null_mut();
-            let _ = ffi::gdk_content_provider_get_value(
-                self.as_ref().to_glib_none().0,
-                value.to_glib_none_mut().0,
-                &mut error,
-            );
-            if error.is_null() {
-                Ok(())
-            } else {
-                Err(from_glib_full(error))
-            }
-        }
-    }
-
     fn ref_formats(&self) -> Option<ContentFormats> {
```

Comment:
as the subclassing code renames those two getters, they should be renamed here as well

Reply:
The same thing for their nullability

Reply:
It conflicts with the property getters. Which one should I ignore?

Reply:
I ignored the property getters.

Reply:
Are the property getters doing exactly the same? Then good that it conflicts ;) You only want one of the two, and the non-property getter is usually the better choice.